src/main/java/org/sagebionetworks/bridge/services/AssessmentConfigService.java

```diff
@@ -53,13 +53,10 @@ AssessmentConfigValidator getValidator() {
         return AssessmentConfigValidator.INSTANCE;
     }
     
-    public AssessmentConfig getAssessmentConfig(String appId, String guid) {
+    public AssessmentConfig getAssessmentConfig(String appId, String ownerId, String guid) {
```

Comment:
Is this being called by the client for a participant, and if so, would this block it?

Reply:
Configs now do not check ownership.